src/Lean/Elab/BuiltinCommand.lean

```diff
@@ -138,6 +203,117 @@ private partial def elabChoiceAux (cmds : Array Syntax) (i : Nat) : CommandElabM
     aliases := aliases.push (currNamespace ++ id, declName)
   modify fun s => { s with env := aliases.foldl (init := s.env) fun env p => addAlias env p.1 p.2 }
 
+/-- Makes names from other namespaces visible without writing the namespace prefix.
+
+Names that are made available with `open` are visible within the current `section` or `namespace`
+block. The `open` command can be used in few different ways:
+
+* `open Some.Namespace.Path1 Some.Namespace.Path2` makes all names in `Some.Namespace.Path1` and
```

Comment:
I think it's clearer to write this up front:
```suggestion
* `open Some.Namespace.Path1 Some.Namespace.Path2` makes all non-`protected` names in `Some.Namespace.Path1` and
```

Reply:
Agreed, done.